tests/vm_test.go

```diff
@@ -451,20 +447,20 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					)).GeneratePayload()
 				Expect(err).ToNot(HaveOccurred())
 
-				vm, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, resourcesPatch, metav1.PatchOptions{})
+				_, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, resourcesPatch, metav1.PatchOptions{})
```

Comment:
Can you help me to understand why this patch is needed, by updating `/spec/template/spec/domain/resources` after stopping the VM?

Reply:
not sure I understand your question..
If is it why change the vm and not just restart then I think its in order to compare the revision and see that it changed

Reply:
Got it, thanks :)